.github/workflows/ci.yml

```diff
@@ -97,14 +97,3 @@ jobs:
   #     - run: npx lerna publish from-package --yes --no-verify-access --canary
   #     - if: always()
   #       run: rm .npmrc
-
-  docs:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v3
-    - name: Setup Python
-      uses: actions/setup-python@v4
-      with:
-        python-version: 3.6
-    - run: pip install -r requirements.docs.txt
-    - run: mkdocs build --clean
```

Comment:
To be able to merge this, maybe make the `docs` step be a no-op, or perhaps just leave it as is since having the ability to latest docs could be useful...